pkg/bpf/bpf_linux.go

```diff
@@ -132,13 +132,12 @@ func OpenOrCreateMap(spec *ebpf.MapSpec, pinDir string) (*ebpf.Map, error) {
 		}
 		defer m.Close()
 
-		log.WithField(logfields.Path, path.Join(pinDir, spec.Name)).
-			WithFields(logrus.Fields{
-				"old": fmt.Sprintf("Type:%s KeySize:%d ValueSize:%d MaxEntries:%d Flags:%d",
-					m.Type(), m.KeySize(), m.ValueSize(), m.MaxEntries(), m.Flags()),
-				"new": fmt.Sprintf("Type:%s KeySize:%d ValueSize:%d MaxEntries:%d Flags:%d",
-					spec.Type, spec.KeySize, spec.ValueSize, spec.MaxEntries, spec.Flags),
-			}).Info("Unpinning map with incompatible properties")
+		logger.Info(
+			"Unpinning map with incompatible properties",
+			logfields.Path, path.Join(pinDir, spec.Name),
+			logfields.Old, m,
```

Comment:
nit: Previous log entry would print the map type, whereas `spec.String()` only does {key,val}size maxEntries and flags. Might be worth including this in case we're ever confused about repinning a map due to a map type change.

Reply:
Oh, good catch!